src/wirecloud/commons/static/js/StyledElements/ModelTable.js

```diff
@@ -485,33 +505,49 @@
 
     ModelTable.prototype.insertInto = function insertInto() {
```

Comment:
This method should be removed from `ModelTable.js` as we are using directly the implementation provided by `StyledElement`.